2select-html/styles.css

```diff
@@ -2,15 +2,47 @@ body {
     font-family: sans-serif;
 }
 
-/* h2 {
+h2, 
+h3,
+ol strong {
     font-weight: normal;
 }
 
-h3 {
+h2 {
+    color: maroon;
+}
+
+/* h3 {
     font-weight: normal;
```

Comment:
You can delete this commented code, as the styling is implemented in the block starting on line 5

Reply:
done! :)